src/layers/conv.jl

```diff
@@ -393,3 +393,49 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", l.aggr)
     print(io, ")")
 end
+
+
+"""
+GINConv([fg,] nn, [eps])
+
+Graph Isomorphism Network
+
+# Arguments
+
+- `fg`: Optionally pass in a FeaturedGraph as input.
+- `nn`: A neural network/layer.
+- `eps`: Weighting factor. Default 0.
+
+"""
+
+struct GINConv{V<:AbstractFeaturedGraph,R<:Real} <: MessagePassing
+    fg::V
+    nn
+    eps::R
+end
+
+function GINConv(fg::V, nn; eps=zero(T)) where {V <: AbstractFeaturedGraph, 
+                                                T <: Real}
+    GINConv(fg, nn, eps)
+end
+
+function GINConv(nn; eps=zero(T)) where {T <: Real}
+    GINConv(NullGraph(), nn, eps)
+end
+
+Flux.trainable(g::GINConv) = (fg=g.fg,nn=g.nn)
+
+message(g::GINConv, x_i::AbstractVector, x_j::AbstractVector) = x_j 
+update(g::GINConv, m::AbstractVector, x) = g.nn((1.0 + g.eps) * x + m)
```

Comment:
```suggestion
update(g::GINConv, m::AbstractVector, x) = g.nn((1 + g.eps) * x + m)
```
otherwise it will promote to Float64

Reply:
Done